azure/ipa.tf

```diff
@@ -238,9 +237,6 @@ resource "helm_release" "ipa-crds" {
     enabled: false
 
   cert-manager:    
-    #dns01RecursiveNameserversOnly: true
```

Comment:
Interesting. I had to make a similar change for private clusters.  Perhaps instead of removing them consider adjusting the variable> https://github.com/IndicoDataSolutions/tf_cod/pull/284

Reply:
Apparently this hasn't been merged in yet?

Maybe we can table this and adjust the code in a future PR?

Reply:
https://indicodata.atlassian.net/browse/DOP-2306